src/pages/workspaces/workspace/Notebooks.js

```diff
@@ -363,7 +361,7 @@ const Notebooks = _.flow(
             tooltip: appIsSettingUp(app) && 'Your Galaxy app is being created',
             onClick: () => setOpenGalaxyConfigDrawer(true)
           }, [
-            getGalaxyText()
+            getGalaxyText(app, galaxyDataDisks)
```

Comment:
This tool tip is using `appIsSettingUp` so displays while app is creating but this button continues to say `create a galaxy app`. Probably should use that `appIsSettingUp` for this Galaxy text as well

Reply:
Thanks for this catch! I changed the logic around and it seems to work as intended ™